gammapy/background/models.py

```diff
@@ -71,3 +77,491 @@ def evaluate(self, x, y):
+
+        Returns
+        -------
+        bg_cube : `~gammapy.models.CubeBackgroundModel
+		bg model cube object
+        """
+        # TODO: should we use the Table class here to read the data?
+        hdu_list = fits.open(filename)
+        hdu = hdu_list['BACKGROUND']
+
+        header = hdu.header
+        data = hdu.data
+
+        det_bins_x = _make_bin_edges_array(data['DETX_LO'], data['DETX_HI'])
+        det_bins_y = _make_bin_edges_array(data['DETX_LO'], data['DETX_HI'])
+        if (det_bins_x == det_bins_y).all():
+            det_bins = det_bins_x
+        else:
+            raise ValueError(
+                "Det bins not matching for x and y ({0}, {1}); is detector not simmetric?".format(
+                    det_bins_x, det_bins_y))
+        if header['TUNIT1'] == header['TUNIT2'] == header['TUNIT3'] == header['TUNIT4']:
+            det_unit = header['TUNIT1']
+        else:
+            raise ValueError("Detector x, y units not matching ({0}, {1}, {2}, {3})"
+                             .format(header['TUNIT1'], header['TUNIT2'], header['TUNIT3'], header['TUNIT4']))
+        #det_bins = Angle(det_bins, 'degree')
+        #det_bins = Angle(det_bins, 'radian')
+        det_bins = Angle(det_bins, det_unit)
+        # TODO: SPATIAL UNITS HARD CODED!!! but detx/y units don't appear in the fits file?!!! -> ~solved (keep reading)
+        #       bei M.Mayer Dateien ja, bei der CTA Beispiel nein (und ich denke es ist in rad)
+        # TODO: edit the example, save it with units, and use it as the test file
+        #       (after checking for consistency with original file)!!!
+
+        energy_bins = _make_bin_edges_array(data['ENERG_LO'], data['ENERG_HI'])
+        if header['TUNIT5'] == header['TUNIT6']:
+            energy_unit = header['TUNIT5']
+        else:
+            raise ValueError("Energy units not matching ({0}, {1})"
+                             .format(header['TUNIT5'], header['TUNIT6']))
+        energy_bins = Quantity(energy_bins, energy_unit)
+        background = data['Bgd'][0]
+        background_unit = header['TUNIT7']
+        if background_unit in ['1/s/TeV/sr']:
+            background_unit = '1 / (s TeV sr)'
+        elif background_unit in ['1/s/MeV/sr']:
+            background_unit = '1 / (s MeV sr)'
+        else:
+            raise ValueError("Cannot interpret units ({})".format(background_unit))
+        background = Quantity(background, background_unit)
+
+        return CubeBackgroundModel(det_bins=det_bins,
+                                   energy_bins=energy_bins,
+                                   background=background)
+
+    @property
+    def image_extent(self):
+        """Image extent `(x_lo, x_hi, y_lo, y_hi)`.
+
+        Returns
+        -------
+        im_extent : `~astropy.coordinates.Angle`
+        	array of bins with the image extent
+        """
+        b = self.det_bins.degree
+        return Angle([b[0], b[-1], b[0], b[-1]], 'degree')
+
```

Comment:
There should only be 1 blank line between methods  - PEP8 will list such things for you.